lib/Github/Api/Repository/Protection.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Github\Api\Repository;
+
+use Github\Api\AbstractApi;
+
+/**
+ * @link   https://developer.github.com/v3/repos/branches/#get-branch-protection
+ * @author Brandon Bloodgood <bbloodgood@gmail.com>
+ */
+class Protection extends AbstractApi
+{
+    public function show($username, $repository, $branch)
```

Comment:
Please add method comments with parameters and their types, link to docs and return value